requirements_dev.in

```diff
@@ -7,4 +7,4 @@ factory_boy
 Faker
 ipdb
 pip-tools
-
+babel
```

Comment:
As said in another comment, see if this is really necessary. If not, delete this dependency